charts/aerospike/templates/configmaps.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aerospike-config
+data:
+  aerospike.conf: |
+    # Aerospike database configuration file
+    # This template sets up a single-node, single namespace developer environment.
+    #
+    # Alternatively, you can pass in your own configuration file.
+    # You can see more examples at
+    # https://github.com/aerospike/aerospike-server/tree/master/as/etc
```

Comment:
Can probably remove this confusing comment

Reply:
Will do.